Sources/BraintreeCore/Public/BraintreeCore/BTAppContextSwitcher.h

```diff
@@ -44,6 +44,15 @@ NS_ASSUME_NONNULL_BEGIN
 */
 @property (nonatomic, copy) NSString *returnURLScheme;
 
+/**
+ The URL scheme provided to the native PayPal Checkout SDK.
+
+ Used as part of the authentication process to identify your application.
+ The value provided here should match one set in the `Return URLs` section of your application's dashboard. You can access the dashboard
+ by [loggin into your PayPal developer account](https://developer.paypal.com)
```

Comment:
I'm not sure all Braintree merchants will have access to this PayPal dev portal. You mentioned there was some API we could hit to set this `returnURL` programmatically instead?